docs/docs/code/python/using-data-stores/README.md

```diff
@@ -0,0 +1,230 @@
+---
+short_description: Store and read data with data stores.
+thumbnail: https://res.cloudinary.com/pipedreamin/image/upload/v1646763735/docs/icons/icons8-database-96_iv1oup.png
+---
+
+# Using Data Stores 
+
```

Comment:
We should probably link to the general data stores docs near the top in case users don't know what they are in general